samples/gen-go-db/server/db/dynamodb/thingwithdatetimecomposite.go

```diff
@@ -160,6 +160,69 @@ func (t ThingWithDateTimeCompositeTable) getThingWithDateTimeCompositesByTypeIDA
 	return decodeThingWithDateTimeComposites(queryOutput.Items)
 }
 
+func (t ThingWithDateTimeCompositeTable) getThingWithDateTimeCompositesByTypeIDAndCreatedResourcePage(ctx context.Context, input db.GetThingWithDateTimeCompositesByTypeIDAndCreatedResourcePageInput, fn func(m *models.ThingWithDateTimeComposite, lastThingWithDateTimeComposite bool) bool) error {
+	queryInput := &dynamodb.QueryInput{
```

Comment:
might be worthwhile to add a check that `StartingAt` and `Limit` (required fields) are non-nil